.yarnrc

```diff
@@ -1,2 +1 @@
---install.frozen-lockfile true
```

Comment:
Yeah, I do that all the time too.

Reply:
Revert, or leave as is?

Reply:
cc @eps1lon

Reply:
What did you need it for?

Reply:
I removed it to upgrade dependency and committed accidentally. Why do you need it? It works better on CI side where lockfile should not change.

Reply:
With this configuration on, you can edit the package.json and expect `yarn` to update the lock file.

Reply:
Lockfile is not changed randomly. What you probably see is the bug with workspaces. add/remove/upgrade commands can't properly regenerate lockfile. Running `yarn` after that solves the problem.

Reply:
Or why forbid to edit package.json? It should be editable in case like this.

Reply:
> With this flag, you can't edit the package.json and expect yarn to update the lock file.

`yarn workspace @material-ui/core add packageName` should work. This worked so far for me.

We can remove this (as well as CI) but it was added to CI because changing github dependencies in CI. If we remove it from the yarnrc then fresh clones might get other node_modules compared to the green build which leads to bad 1st contributor experience. 

IMO it should be expected that manually editing package(lock) files takes some more work. That's what the CLI is for.